src/config/file_info.rs

```diff
@@ -150,13 +163,19 @@ impl FileInfo<'_, '_, '_, '_> {
         if let Some(mode) = self.mode {
             rpm_file_option = rpm_file_option.mode(mode as i32);
         }
+        if let Some(caps) = self.caps {
+            rpm_file_option = match rpm_file_option.caps(caps) {
+                Ok(v) => v,
+                Err(_) => return Err(ConfigError::WrongType("caps".to_string(), "capability")),
+            };
+        }
```

Comment:
`ConfigError::WrongType` is not appropriate for this purpose.
Please add a new error expressing `rpm::Error` on a per asset file basis.

It would be as following code:
```rust
rpm_file_option = rpm_file_option.caps(caps).map_err(|err| ConfigError::AssetFileRpm(idx, "caps", err))?;
```

Reply:
That is sensible I have updated. 
Unfortunately I wasn't able to directly re-use the `Error` from rpm-rs as it doesn't implement `Clone`.

Reply:
@dsteeley How about using reference-counting pointer?

```patch
     #[error("Field {1} for file {0} has the following error: {2}")]
-    AssetFileRpm(usize, &'static str, String),
+    AssetFileRpm(usize, &'static str, #[source] std::rc::Rc<rpm::Error>),
```

Reply:
That's a solution I wasn't aware of, thanks for the guidance. 
I've made that change.